test/unit/storage/accessors/PassthroughDataAccessor.test.ts

```diff
@@ -0,0 +1,87 @@
+import { BasicRepresentation } from '../../../../src/http/representation/BasicRepresentation';
+import { RepresentationMetadata } from '../../../../src/http/representation/RepresentationMetadata';
+import type { DataAccessor } from '../../../../src/storage/accessors/DataAccessor';
+import { PassthroughDataAccessor } from '../../../../src/storage/accessors/PassthroughDataAccessor';
+import { guardedStreamFrom } from '../../../../src/util/StreamUtil';
+
+describe('ValidatingDataAccessor', (): void => {
+  let passthrough: PassthroughDataAccessor;
+  let childAccessor: jest.Mocked<DataAccessor>;
+
+  const mockIdentifier = { path: 'http://localhost/test.txt' };
+  const mockMetadata = new RepresentationMetadata();
+  const mockData = guardedStreamFrom('test string');
+  const mockRepresentation = new BasicRepresentation(mockData, mockMetadata);
+
+  beforeEach(async(): Promise<void> => {
+    jest.clearAllMocks();
+    childAccessor = {
+      canHandle: jest.fn(),
+      writeDocument: jest.fn(),
+      getData: jest.fn(),
+      getChildren: jest.fn(),
+      writeContainer: jest.fn(),
+      deleteResource: jest.fn(),
+      getMetadata: jest.fn(),
+    };
+    childAccessor.getChildren = jest.fn();
+    passthrough = new PassthroughDataAccessor(childAccessor);
+  });
+
+  describe('writeDocument()', (): void => {
+    it('should call the accessors writeDocument() function.', async(): Promise<void> => {
+      const spy = jest.spyOn(childAccessor, 'writeDocument');
+      await passthrough.writeDocument(mockIdentifier, mockData, mockMetadata);
+      expect(spy).toHaveBeenCalledTimes(1);
+      expect(spy).toHaveBeenCalledWith(mockIdentifier, mockData, mockMetadata);
```

Comment:
While you can use a spy, you don't actually need to since you already defined the functions to be jest functions, so you can also just do the following:
```suggestion
      await passthrough.writeDocument(mockIdentifier, mockData, mockMetadata);
      expect(childAccessor.writeDocument).toHaveBeenCalledTimes(1);
      expect(childAccessor.writeDocument).toHaveBeenCalledWith(mockIdentifier, mockData, mockMetadata);
```

You can do this everywhere where you're using spies on jest functions.